docs/features/dispenser.md

```diff
@@ -0,0 +1,81 @@
+# AlgoKit Dispenser
```

Comment:
Should we call it AlgoKit TestNet Dispenser for clarity?

Reply:
Although on reflection in the future we could plumb it to KMD for localnet dispensing too I guess?

Reply:
@robdmoore  Sure i can rename to AlgoKit TestNet Dispenser for clarity but a bit confused on what you mean by `plumbing to kmd for localnet dispensing`? Can you expand on this a bit?